client/src/Atoms/Rocket/RocketStyled.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const RocketStyled = styled.div`
+  transform: translate(0%);
```

Comment:
You don't need line 4 and 5.

Reply:
I need those. :) They provide a smooth animation on mouse leave